uportal-war/src/main/resources/layout/theme/respondr/content.xsl

```diff
@@ -411,7 +411,8 @@
       </xsl:if>
 
       <!-- Remove Icon -->
-      <xsl:if test="@unremovable='false' and not(//focused) and /layout/navigation/tab[@activeTab='true']/@immutable='false'">
+      <!-- note: deleteAllowed will either be false or not present if set from the admin ui -->
+      <xsl:if test="(@unremovable='false' and not(@dlm:deleteAllowed='false')) and not(//focused) and /layout/navigation/tab[@activeTab='true']/@immutable='false'">
```

Comment:
I verified there is no other reference in theme/*/*.xsl to that attribute so I also suggest removing it to avoid confusion.  I updated https://wiki.jasig.org/display/UPM41/Setup+a+fragment+layout+by+modifying+fragment-layout+XML+files indicating it is obsolete as well.  There are some code references to it, but they should be deleted.  I see you created https://issues.jasig.org/browse/UP-4399 so I updated it to also remove "immutable"